package.json

```diff
@@ -16,15 +16,14 @@
   },
   "homepage": "https://github.com/ralphtheninja/pgdown#readme",
   "dependencies": {
-    "abstract-leveldown": "~2.4.1",
```

Comment:
Nope -- that just inherits from `abstract-leveldown`, but allows you to set a `_createReadStream` rather than an iterator (`pg-cursor` was giving me some grief, though I think I figured it out).

What's particularly nice is that this will allow me to turn the batch stuff into a beautifully simple `_createWriteStream` and `abstract-stream-leveldown` will hook us up w/ `batch` and `chainedBatch`. This is a version of the elegant two-method api I've been hoping for.